src/utils/http-validators.ts

```diff
@@ -0,0 +1,12 @@
+import { BadRequestHttpError, HttpHandlerContext } from "@digita-ai/handlersjs-http";
+
+export const validateContentType = (ctx: HttpHandlerContext, contentType: string): void => {
+  if (
+    ctx.request.headers['content-type'] &&
+    ctx.request.headers['content-type'].toLowerCase() === contentType.toLowerCase()
```

Comment:
```suggestion
    ctx.request.headers['content-type']?.startsWith(contentType.toLowerCase())
```

I'll double-check it tomorrow.

* equality is too strict, it should match values with extra parameter like common: `application/json; charset=UTF-8`
* I think hanldersjs already lowercase the headers